pentasharp/Controllers/TransportController.cs

```diff
@@ -0,0 +1,19 @@
+using Microsoft.AspNetCore.Mvc;
+using System.Diagnostics;
+using WebApplication1.Models;
+
+namespace WebApplication1.Controllers
+{
+    public class TransportController : Controller
+    {
+        public IActionResult BusSchedule()
+        {
+            return View();
+        }
+
```

Comment:
Same as per DashboardController, separate BusSchedule and Taxi. I dont quite like Taxis as a naming here.